src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -459,8 +445,10 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter {
             return true;
         }
 
-        private CacheSettings CacheSettings {
-            get {
+        private CacheSettings CacheSettings
+        {
+            get
+            {

```

Comment:
Why this rebracing? Do we have one style for methods and another for properties?